packages/zmarkdown/config/remark.js

```diff
@@ -29,6 +29,7 @@ const remarkConfig = {
     blocks: [],
   },
 
+  canUseTextr: true,
```

Comment:
I'd prefer something like `useTextr` or `enableTextr` or (with the value as `false`) `disableTextr`. It's not that this option let's us decide later whether we want to use textr or not, this option explicitly enables it.